src/garage/np/exploration_policies/add_gaussian_noise.py

```diff
@@ -81,3 +82,36 @@ def get_actions(self, observations):
         return np.clip(actions + np.random.normal(size=actions.shape) * sigma,
                        self._action_space.low,
                        self._action_space.high), agent_infos
+
+    def update(self, episode_batch):
+        """Update the exploration policy using a batch of trajectories.
+
+        Args:
+            episode_batch (EpisodeBatch): A batch of trajectories which
+                were sampled with this policy active.
+
+        """
+        self._episode_num += episode_batch.lengths.shape[0]
+        self._iteration = self._episode_num
+
+    def get_param_values(self):
+        """Get parameter values.
+
+        Returns:
+            list or dict: Values of each parameter.
+
+        """
+        return {
+            'episode_num': self._episode_num,
+            'inner_params': self.policy.get_param_values()
+        }
+
+    def set_param_values(self, params):
+        """Set param values.
+
+        Args:
+            params (np.ndarray): A numpy array of parameter values.
+
+        """
+        self._episode_num = params['episode_num']
```

Comment:
Please merge `episode_num` and `iteration`. Since you don't set `iteration` here, this change is probably broken. Please write a test to confirm that the noise level is actually changed by `set_param_values`